#
# Results of raycasting algae at several pixels
#

# Whether an alga has been hit
bool alga_hit

# Pixels information
PixelRay[] pxl_info
